github/messages.go

```diff
@@ -130,6 +130,8 @@ func messageMAC(signature string) ([]byte, func() hash.Hash, error) {
 // The Content-Type header of the payload can be "application/json" or "application/x-www-form-urlencoded".
 // If the Content-Type is neither then an error is returned.
 // secretKey is the GitHub Webhook secret message.
+// If your webhook does not contain a secret message, you can pass nil or an empty slice.
+// This is intended for local development purposes only and all webhooks should contain a secret message.
```

Comment:
minor nit: I believe we should maintain semantic consistency between GitHub documentation and the library's documentation so that the user can easily navigate between both of them with ease. Thus, I request that we update this to read `"... all webhooks should ideally set up a secret token."`

Thoughts @gmlewis @jbrockopp

Reply:
Sounds great!  Let's please go ahead and address this change in this PR.

Reply:
@kshitij10496 @gmlewis sure I can do that. I was just following the previously used verbiage in the existing code.

> // secretKey is the GitHub Webhook secret message.